packages/zoe/src/zoeService/types.js

```diff
@@ -33,7 +33,6 @@
  * @property {GetBrands} getBrands
  * @property {import('./utils').GetTerms} getTerms
  * @property {GetOfferFilter} getOfferFilter
- * @property {SetOfferFilter} setOfferFilter
```

Comment:
absent? We do have a set offer filter thing somewhere, I'm pretty sure... Where did it go?

_I can look it up, but maybe you know offhand._

Reply:
Both set and get are on the ZCF (runtime and static types). `getOfferFilter` is ZoeService at runtime and `setOfferFilter` isn't. I'm not sure why it seemed intentional. @Chris-Hibbert ?

Reply:
OK, on review of stuff like [governFilter.js](https://github.com/Agoric/agoric-sdk/blob/master/packages/governance/src/contractGovernance/governFilter.js), I see that yes, this is by design:

 - offer filter governance is done via zcf; that is: by voluntary compliance of the contract
 - zoe (a ZoeService) is widely held. If it offered `setOfferFilter` to its clients then anyone could filter offers on any contract